about.html

```diff
@@ -20,20 +20,20 @@
 
   <div class="content-parsys responsivegrid">
     <div class="aem-Grid aem-Grid--12 aem-Grid--default--12 ">
-        <!-- HEADER WRAP -->
+        <!-- HEADING WRAP -->
         <div class="container responsivegrid header aem-GridColumn aem-GridColumn--default--12">
           <div id="header-main" class="cmp-container">
             <div class="aem-Grid aem-Grid--12 aem-Grid--default--12 ">
               <!-- SEARCH OVERLAY WRAP -->
               <div class="container responsivegrid home-search-overlay aem-GridColumn aem-GridColumn--default--12 layout-grid">
                 <div id="search-overlay" class="cmp-container">
                   <!-- CLOSE IMAGE -->
-                  <div class="close">
-                    <div class="cmp-image">
-                      <a href="#" class="close">
-                        <img class="cmp-image__image" src="./assets/close.svg">
-                      </a>
-                    </div>
+                  <div class="button image close">
+                    <button type="button" id="button-5487aab983" class="cmp-button" data-cmp-clickable="" data-cmp-data-layer="{&quot;button-5487aab983&quot;:{&quot;@type&quot;:&quot;core-components-examples/components/button&quot;,&quot;dc:title&quot;:&quot;Button&quot;}}">
+                      <span class="cmp-button__icon cmp-button__icon--close close" aria-hidden="true">
+                        <img src="./assets/close.svg" class="cmp-image-button__icon" itemprop="contentUrl" data-cmp-hook-image="image" alt="Close">
```

Comment:
@Prystal254 Button components don't use `<img />` for the icons. You need to use a pseudo element.